olm_deploy/scripts/operator-install.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 set -eou pipefail
+set -x
```

Comment:
this tends to make for hideous output.  Do we really need to commit it?

Elsewhere we usually wrap this in an if $DEBUG block so you can turn it on when testing locally.  (Of course there is no easy way to turn it on in a CI run on your PR, but in that case you can make the temporary change in your PR while you are debugging, also).

Reply:
No, fully agreed, embarassing oversight. ALready fixed.